webpack.common.js

```diff
@@ -205,6 +208,7 @@ const config = {
                     path.resolve(__dirname, 'node_modules/markdown-it'),
                     path.resolve(__dirname, 'node_modules/material-react-table'),
                     path.resolve(__dirname, 'node_modules/mdurl'),
+                    path.resolve(__dirname, 'node_modules/pdfjs-dist'),
```

Comment:
I didn't notice any issues moving this back into our standard babel build, but is there anything specific that needs tested here @dmitrylyzo?

Reply:
Tbh, I don't remember. :sweat_smile:
Maybe it's specific to that version of `pdfjs`?

At the moment, webOS 1.2 fails with `SyntaxError: Use of reserved word 'import'` (also tried it in the extra babel list).
```js
case 2:
  _context5.next = 4;
  return import(/*webpackIgnore: true*/_this37.workerSrc);
case 4:
```
_I'm using `source-map` in `webpack.dev.conf`, so it doesn't produce `eval`, which doesn't work in webOS 1.2._

Anyway, `pdf.worker` doesn't work because of ES6 features.
But it looks like `pdfjs` has some legacy variant (`legacy/` folder).
No, that doesn't work either because of `import`. :confused: